conda_forge_tick/update_upstream_versions.py

```diff
@@ -71,16 +63,27 @@ def ignore_version(attrs: Mapping[str, Any], version: str) -> bool:
     )
 
 
-def get_latest_version(
+def get_latest_version_local(
     name: str,
     attrs: Mapping[str, Any],
     sources: Iterable[AbstractSource],
 ) -> Dict[str, Union[Literal[False], str]]:
     """
     Given a package, return the new version information to be written into the cf-graph.
-    :param name: the name of the package.
-    :param attrs: the node attributes of the package
-    :param sources: the version sources to use (sources can be excluded by the package but not added)
+
+    Parameters
+    ----------
+    name : str
+        The name of the feedstock.
+    attrs : Mapping[str, Any]
+        The node attributes of the feedstock.
+    sources : Iterable[AbstractSource]
+        The version sources to use.
```

Comment:
Personally, if the signature is already typed, I'm not adding that info in the docstring.

Reply:
hahahaha - this was all github copilot! I let it add things at ~random and double check they don't look crazy. :p